### PR TITLE
Adding two blog posts

### PR DESCRIPTION
**Pull request by zohamid**
Blog posts dated 11th July and 22nd July were added. 
Titled - (1) Intro to PyData Conf 2017 and (2) Why be a PyData member.

### CODE REVIEW

**Comment by Dawny33** [comment]
LGTM. 

cc @manojpandey

File: _posts/2017-07-11-PyDataDelhi Conference 2017.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+And this time, PyData Delhi chapter is proud to hold it's very first conference at IIIT-Delhi. Woot woot! Looking forward to seeing young bright minds and seasoned programmers alike, with us on 2nd and 3rd of September!
+
+*PyData is an educational program of NumFOCUS, a 501(c)3 non-profit organization in the United States.*
+![PyData Delhi](/home/zoha/Desktop/logo.png  "PyData Delhi")
+*Women Who Code is a 501(c)(3) not-for-profit organization*
+![Women Who Code, Delhi](/home/zoha/Desktop/wwc/WWC.png  "Women Who Code, Delhi")
+
+- Zoha Hamid, Member, PyData Delhi; Key Team member, WWC Delhi
```
Comment (manojpandey) [inline _posts/2017-07-11-PyDataDelhi Conference 2017.md:38]: Can we change this to something like:

> written by @zohamid 
on behalf of PyData Delhi & WWC Delhi Team

File: _posts/2017-07-22-Why be a part of PyData.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+There are a numerous other reasons, including how PyData helps me feel about giving back to my local community and supporting diverse initiatives in technical communities.
+Infact, being an organiser for PyData Delhi Conf'17 has given me a high and a rush like no other. I feel proud to be a part of such an active community and give back to the community by helping my peers and introducing them to open-source. The experience with my fellow Organising Team members has been amazing so far and we can't wait for September to arrive!
+So whatever your personal reasons maybe, I hope that listing down a few of mine have given you something to dwell upon. Go on, join a community near you and be a part of change and growth.
+Perhaps, it has motivated you to attend the upcoming PyData Delhi conference [PyData Delhi conference](https://pydata.org/delhi2017/) ! Ciao, and see you there folks!
+
+Signing off,
+
+__Zoha Hamid__
```
Comment (manojpandey) [inline _posts/2017-07-22-Why be a part of PyData.md:30]: Again, can we change this to something like:

> written by @zohamid 
on behalf of PyData Delhi & WWC Delhi Team

Comment (zohamid) [reply _posts/2017-07-22-Why be a part of PyData.md:30]: Sure, that will work as well.

**Pull request merged**